.env.example

```
@@ -3,11 +3,9 @@ ROOT_DOMAIN=example.com
 STACK_VERSION=1
 
 ## App
-DASHBOARD_AUTH="admin:admin"
-DASHBOARD_HOST="admin.example.com"
-
 BACKEND_HOST="example.com"
 
 ## Monitoring
 PROMETHEUS__IP_ALLOWLIST=10.0.0.0/8,172.16.0.0/12
 ALERTMANAGER__IP_ALLOWLIST=10.0.0.0/8,172.16.0.0/12
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALERTMANAGER__IP_ALLOWLIST key should go before the PROMETHEUS__IP_ALLOWLIST key

(UnorderedKey)